tests/CMakeLists.txt

```diff
@@ -78,21 +17,27 @@ FetchContent_Declare(
 set(gtest_force_shared_crt ON CACHE BOOL "" FORCE)
 FetchContent_MakeAvailable(googletest)
 
+enable_testing()
+add_executable( polygames-tests
+ # Include your tests here.
+ connectfour-tests.cc
+ havannah-state-tests.cc
+ havannah-tests.cc
+ hex-state-tests.cc
+ hex-tests.cc
+ test_state.cc
+ test_mcts.cc
 
-target_include_directories(polygames-tests PUBLIC ${PYTHON_INCLUDE_DIRS})
+ # TODO: add and fix tests
```

Comment:
is there a reason to delete this TODO? Seems good to remember to get those tests working once we start using ludii?

Reply:
wait, I didn't delete this TODO, this is a line starting with `+` and in green.

Reply:
ofc, makes sense, thanks